prboom2/src/p_enemy.c

```diff
@@ -2964,6 +2964,104 @@ void A_LineEffect(mobj_t *mo)
   mo->player = oldplayer;
 }
 
+//
+// [XA] New mbf21 codepointers
+//
+
+//
+// A_SpawnFacing
+// Spawns an actor facing the same direction as the caller.
+// Basically just A_Spawn with a major quality-of-life tweak.
+//   misc1: Type of actor to spawn
+//   misc2: Height to spawn at, relative to calling actor
+//
+void A_SpawnFacing(mobj_t *actor)
+{
+  if (!mbf21 || !actor->state->misc1)
+    return;
+
+  mobj_t *mo = P_SpawnMobj(actor->x, actor->y, (actor->state->misc2 << FRACBITS) + actor->z, actor->state->misc1 - 1);
```

Comment:
We need to declare this variable at the top and then set it here to avoid a warning in C, right? 🤓

Reply:
Heh, maybe. :P -- Fixed for consistency at least. ;)